config/custom-formats.ts

```diff
@@ -0,0 +1,8 @@
+// Note: This is the list of formats
+// The rules that formats use are stored in data/rulesets.ts
+
+// easy to read format list type
+export type FormatList = (FormatData | {section: string, column?: number});
```

Comment:
I don't think this is the correct place to define this type, I would check out `sim/global-types.ts`

Reply:
Good point, I will move it there